docs/basics/101-130-yodaproject.rst

```diff
@@ -643,7 +643,8 @@ configure this repository as a sibling of the dataset:
     .. runrecord:: _examples/DL-101-130-116
        :language: console
        :workdir: dl-101/DataLad-101/midterm_project
-       :realcommand: datalad --log-level critical siblings add -d . --name github --url https://github.com/adswa/midtermproject.git
+       :realcommand: python -c "from datalad.core.distributed.tests.test_push import mk_push_target; from datalad.api import Dataset as ds; mk_push_target(ds=ds('/home/me/dl-101/DataLad-101/midterm_project'), name='github', path='/home/me/pushes/midtermproject', annex=False, bare=True)"
```

Comment:
This will be done frequently. I guess we should have a helper command or shell alias that reduces this to the essential parameters.

Reply:
I have drafted an addition to ``autorunrecord``: It adds an additional option ``:makepushtarget:`` and can take a dictionary with command specifications. Will PR this shortly.

Reply:
https://github.com/mih/autorunrecord/pull/6